apps/meteor/app/threads/server/methods/followMessage.ts

```diff
@@ -43,6 +44,10 @@ Meteor.methods<ServerMethods>({
 
 		const followResult = await follow({ tmid: message.tmid || message._id, uid });
 
+		void notifyOnMessageChange({
+			id: mid,
```

Comment:
Just realized that the method doesn't necessarily modify the same message that is passed as argument, but your code was sending the event with the id from the params directly.

```suggestion
			id: message.tmid || message._id,
```

Reply:
Nice Catch! Fixed.